client/my-sites/customer-home/quick-links/index.jsx

```diff
@@ -0,0 +1,333 @@
+
+export const QuickLinks = ( {
+	customizeUrl,
+	isStaticHomePage,
+	showCustomizer,
+	hasCustomDomain,
+	menusUrl,
+	editHomepageAction,
+	writePostAction,
+	addPageAction,
+	manageCommentsAction,
+	trackEditMenusAction,
+	trackCustomizeThemeAction,
+	changeThemeAction,
+	trackDesignLogoAction,
+	addEmailAction,
+	addDomainAction,
+} ) => {
+	const translate = useTranslate();
+
+	const quickLinks = (
+		<div className="quick-links__boxes">
+			{ isStaticHomePage ? (
+				<ActionBox
+					onClick={ editHomepageAction }
+					label={ translate( 'Edit homepage' ) }
+					iconSrc={ imagesIcon }
+				/>
+			) : (
+				<ActionBox
+					onClick={ writePostAction }
+					label={ translate( 'Write blog post' ) }
+					iconSrc={ postIcon }
+				/>
+			) }
+			{ isStaticHomePage ? (
+				<ActionBox
+					onClick={ addPageAction }
+					label={ translate( 'Add a page' ) }
+					iconSrc={ pageIcon }
+				/>
+			) : (
+				<ActionBox
+					onClick={ manageCommentsAction }
+					label={ translate( 'Manage comments' ) }
+					iconSrc={ commentIcon }
+				/>
+			) }
+			{ isStaticHomePage ? (
+				<ActionBox
+					onClick={ writePostAction }
+					label={ translate( 'Write blog post' ) }
+					iconSrc={ postIcon }
+				/>
+			) : (
+				<ActionBox
+					onClick={ addPageAction }
+					label={ translate( 'Add a page' ) }
+					iconSrc={ pageIcon }
+				/>
+			) }
+			{ showCustomizer && (
+				<ActionBox
+					href={ menusUrl }
+					onClick={ trackEditMenusAction }
+					label={ translate( 'Edit menus' ) }
+					iconSrc={ menuIcon }
+				/>
+			) }
+			{ showCustomizer && (
+				<ActionBox
+					href={ customizeUrl }
+					onClick={ trackCustomizeThemeAction }
+					label={ translate( 'Customize theme' ) }
+					iconSrc={ customizeIcon }
+				/>
+			) }
+			<ActionBox
+				onClick={ changeThemeAction }
+				label={ translate( 'Change theme' ) }
+				iconSrc={ themeIcon }
+			/>
+			<ActionBox
+				href="https://wp.me/logo-maker"
+				onClick={ trackDesignLogoAction }
+				target="_blank"
+				label={ translate( 'Create a logo with Looka' ) }
+				external
+				iconSrc={ logoIcon }
+			/>
+			{ hasCustomDomain ? (
+				<ActionBox
+					onClick={ addEmailAction }
+					label={ translate( 'Add email' ) }
+					iconSrc={ gSuiteIcon }
+				/>
+			) : (
+				<ActionBox
+					onClick={ addDomainAction }
+					label={ translate( 'Add a domain' ) }
+					iconSrc={ customDomainIcon }
+				/>
+			) }
+		</div>
+	);
+
+	if ( ! isMobile() ) {
+		return (
+			<Card className="quick-links">
+				<CardHeading>{ translate( 'Quick Links' ) }</CardHeading>
+				{ quickLinks }
+			</Card>
+		);
+	}
+	return (
+		<FoldableCard
+			className="quick-links card-heading-21"
+			header={ translate( 'Quick Links' ) }
+			expanded
+		>
+			{ quickLinks }
+		</FoldableCard>
+	);
+};
+
+const editHomepageAction = ( editHomePageUrl, isStaticHomePage ) =>
+	withAnalytics(
+		composeAnalytics(
+			recordTracksEvent( `calypso_customer_home_my_site_edit_homepage_click`, {
```

Comment:
Minor (and non-blocking): 

```suggestion
			recordTracksEvent( 'calypso_customer_home_my_site_edit_homepage_click', {
```

There are more occurrences below (likely missed during copy&paste).

Reply:
Thanks, I'll tidy these!

Reply:
👍 updated in 041e99b7b2a67c92a9984895d24a15a293430de8